main_penalty_calculate/serializers/output_serializer.py

```diff
@@ -3,13 +3,15 @@
 
 class OutputSerializer:
     _OFFENDER_DATA = (
-        '{violation.identity_card_number}; '
-        '{violation.identity_card_name}; '
+        '{identity_card.number}; '
+        '{identity_card.name}; '
         '{license_plate_numbers}; '
-        '{violation.demerit_points}'
+        '{demerit_points}; '
+        '{penalty_amount:.2f}'
     )
     _PATTERN_FOR_REGEX = r'\[|\]|\''
     _EMPTY_STRING = ''
+    _LINE_BREAK = '\n'
```

Comment:
Um comentario para esses nomes, vc nao deve pensar no que eh o valor da variavel e sim pra que ela serve.
Um exemplo eh vc chamar uma variavel de `number = 10` nao ajuda muito a dizer pra que ela serve, mas se o nome for `days_til_expiration = 10` ajuda.

Reply:
Melhorou muito, boa